Alpaca.Markets/Messages/JsonStreamQuote.cs

```diff
@@ -33,16 +33,18 @@ internal class JsonStreamQuote : IStreamQuote
         [JsonIgnore]
         public DateTime Time { get; set; }
 
+        /// <summary>
+        /// Called when the json message is deserialized.
+        /// We adjust the milliseconds by deviding it by 10,000 to reduce the time stamp
+        /// </summary>
+        /// <param name="context">The context.</param>
         [OnDeserialized]
-        internal void OnDeserializedMethod(
-            StreamingContext context)
+        internal void OnDeserializedMethod(StreamingContext context)
         {
 #if NET45
-            Time = DateTimeHelper.FromUnixTimeMilliseconds(Timestamp);
+            Time = DateTimeHelper.FromUnixTimeMilliseconds((Int64)(Timestamp / 10000));
```

Comment:
Shouldn't this be divide by 1000000? I'm pretty sure we're going from nanoseconds -> milliseconds

Reply:
You are right, I missed a couple of zeros on that one. Great catch, will fix and resubmit